components/tools/OmeroWeb/omeroweb/feedback/templates/disabled.html

```diff
@@ -0,0 +1,32 @@
+{% extends "webgateway/core_html.html" %}
+{% load i18n %}
+
+{% block link %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static "feedback/css/layout.css" %}" type="text/css" />
+{% endblock %}
+
+{% block title %}
+    {% trans "OMERO.web - support" %}
+{% endblock %}
+
+{% block head %}
+    {{ block.super }}
+    {% include "webgateway/base/includes/shortcut_icon.html" %}
+{% endblock %}
+
+{% block body %}
+<div class="bodyWrapper">
+    <div><h1>
+        <a href="#" onClick="window.close()">{% trans "Close" %}</a>
```

Comment:
Unclear whether we would like to have a "Go back to OMERO.web" test instead or in addition like in the base template.

Reply:
It's a popup window, so no need for a "go back" link. This is the same as when feedback is enabled.

Reply:
Understood. This is probably an artifact of my testing where I entered the URLs directly (notably to generate the 500 error page).